README.adoc

```diff
@@ -271,20 +367,91 @@ system-deployment-694c7b74f7-lrlf7     0/1       Running   0          1m
+Navigate to `{system-api}/properties`.
+Observe that your request will still be successful because you have two replicas and one is still healthy.
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+You will notice that one of the two **system** pods is no longer in the ready state.
+Make a request to the **/system/properties** endpoint with the following command:
+```
+SYSTEM_NODEPORT=`kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services system-service`
+curl -s http://localhost:$SYSTEM_NODEPORT/system/properties | jq
+```
+{: codeblock}
+
+Observe that your request will still be successful because you have two replicas and one is still healthy.
+endif::[]
 
 === Observing the effects on the inventory microservice
 
-Wait until the `system` pod is ready again. Make two POST requests to `{system-api}/unhealthy`.
- If you see the same pod name twice, make the request again until you see that the second 
- pod has been made unhealthy. You may see the same pod twice because there's a delay 
- between a pod becoming unhealthy and the readiness probe noticing it. Therefore, traffic 
- may still be routed to the unhealthy service for approximately 5 seconds. Continue to 
- observe the output of `kubectl get pods`. You will see both pods are no longer ready. 
- During this process, the readiness probe for the `inventory` microservice will also fail. 
- Observe it's no longer in the ready state either.
+// Static guide instruction
+ifndef::cloud-hosted[]
+Wait until the `system` pod is ready again.
+Make two POST requests to `{system-api}/unhealthy`.
+If you see the same pod name twice, make the request again until you see that the second 
+pod has been made unhealthy. You may see the same pod twice because there's a delay 
+between a pod becoming unhealthy and the readiness probe noticing it.
+Therefore, traffic may still be routed to the unhealthy service for approximately 5 seconds.
+Continue to observe the output of `kubectl get pods`.
+You will see both pods are no longer ready. 
+During this process, the readiness probe for the `inventory` microservice will also fail. 
+Observe it's no longer in the ready state either.
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+Wait until the **system-service** pod is ready again.
+Make two POST requests to the **/system/unhealthy** endpoint of the two **system-service** pods.
+
+Go to the terminal where you started the port forwarding for the **system** service.
+Press **CTRL+C** to stop the port forwarding.
+Run following commands to set up the port forwarding to access the first pod.
+```
+SYSTEM_NODEPORT=`kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services system-service`
+SYSTEM_POD=`kubectl get pods -o=name | grep system | head -1`
+kubectl port-forward $SYSTEM_POD $SYSTEM_NODEPORT:9080
+```
+{: codeblock}
+
+On a terminal, run the following POST request to make the first pod to not ready state. 
```

Comment:
From a terminal, run the following POST request to change the first pod to  a not-ready state.